client/src/pages/Login/Login.tsx

```diff
@@ -20,6 +20,7 @@ export default function Login(): JSX.Element {
         setSubmitting(false);
         updateSnackBarMessage(data.error.message);
       } else if (data.success) {
+        window.location.reload();
```

Comment:
Let's find another way rather than reloading the entire page

Ditto where applicable